### PR TITLE
Create nut.rst

### PR DESCRIPTION
**Pull request by Melantrix**
There is almost no documentation about how to configure NUT. I needed some guidance on IRC to get it working because of that. To fix the real issue here (no documentation) this page is created to have at least a starting point on configuring NUT.

This is my first pull request and documentation page, so please let me know if i need to change anything, including following a different process or something!

### CODE REVIEW

File: source/manual/how-tos/nut.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+--------------------------
+What is NUT
+--------------------------
+
+NUT is a tool that enables the use of a UPS device via either USB or Network communication. 
+
+--------------------------
+Installation
+--------------------------
+NUT is available as a plugin in OPNSense via the plugin **os-nut**. You can install the plugin via **System > Firmware > Plugins > os-nut**. After installation NUT will be available as a service, you may need to login again to make it visible.
```
Comment (mimugmail) [inline source/manual/how-tos/nut.rst:14]: OPNsense, I'd remove the second sentence. Usually a refresh is enough.

File: source/manual/how-tos/nut.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+--------------------------
+How to Setup - Standalone
+--------------------------
+
+*Please add standalone documentation if you have configured this yourself.*
```
Comment (mimugmail) [inline source/manual/how-tos/nut.rst:20]: Can you remove this section? We can't publish this unfinished. I'll take care of the rest ...

File: source/manual/how-tos/nut.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+3. Fill in these values, then click apply:
+
+==================== ============================================
+Enable                Checked
+IP Address            <ip of NUT Server>
+Username              <username, if applicable>
+Password              <password, if applicable>
+==================== ============================================
+
+4. Click on **Diagnostics** in the left menubar. If configured correctly this page will show, after waiting a few seconds, information about the UPS. Loook for **device.mfr** and **device.model** to check if the correct UPS is detected.
```
Comment (mimugmail) [inline source/manual/how-tos/nut.rst:47]: Maybe you can also add a screenshot for this?

**Review by mimugmail — CHANGES_REQUESTED** [review]
Small things, thx for starting this!

File: source/manual/how-tos/nut.rst (hunk 1 above)
Comment (Melantrix) [reply source/manual/how-tos/nut.rst]: Done!

**Review by Melantrix — COMMENTED** [review]
I've changed the file like you suggested, thanks for reviewing!

**Comment by AdSchellevis** [comment]
@Melantrix thanks! pulling this state in now, if you want to extend it later, just open a new PR

**Pull request merged**